src/Symfony/Component/Form/Test/FormIntegrationTestCase.php

```diff
@@ -35,6 +37,11 @@ protected function setUp()
             ->getFormFactory();
     }
 
+    private function doTearDown()
```

Comment:
let's ship this and doSetUp in the traits!

Reply:
I'm not sure to get you.... "ship" what?

Reply:
copy that exact code in the trait - that'll provide a default dummy implementation - then ppl will overwrite when needed

Reply:
But people can overwrite the "real" methods, e.g. setUp and tearDown. No?

Reply:
Sure, but they might have the same issue on their side, this trait can help eg bundle/lib authors to provide auto compat with php8. Like here, they may need it only for setup  but not teardown. Prodiving doTearDown and doSetUp in the trait would prevent all using the trait seamlessly.